test/numpy.test.ts

```diff
@@ -327,6 +327,53 @@ suite.each(devices)("device:%s", (device) => {
     });
   });
 
+  suite("jax.numpy.less()", () => {
+    test("computes less", () => {
+      const x = np.array([1, 2, 3, 4]);
+      const y = np.array([4, 5, 3, 2]);
+      expect(np.less(x.ref, y.ref).js()).toEqual([true, true, false, false]);
```

Comment:
```suggestion
      expect(np.less(x, y).js()).toEqual([true, true, false, false]);
```

Can you please make all the tests take `x` and `y` instead of `x.ref` and `y.ref`? This way x and y will be freed.